deployments/ansible/roles/collector/README.md

```diff
@@ -253,3 +253,126 @@ after installation/configuration in order for any change to take effect.
 
 - `splunk_otel_auto_instrumentation_enable_metrics` (Linux only): Enable or
   disable exporting Micrometer metrics. (**default**: `false`)
+
+### Auto Instrumentation for .NET on Windows
+
+***Warning:*** The `Environment` property in the
+`HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\W3SVC` registry key will
+be overwritten by the options specified below to enable/configure auto
+instrumentation for IIS. Use the
+`signalfx_dotnet_auto_instrumentation_additional_options` option (see below for
+details) to include any other environment variables required for IIS.
+
+**Note:** By default, IIS will be restarted with the `iisreset` command (if it
+exists) after installation/configuration. Applications ***not*** running within
+IIS need to be restarted/managed separately in order for any changes to take
+effect.
+
+For proxy options, see the [Windows Proxy](#windows-proxy) section.
+
+- `install_signalfx_dotnet_auto_instrumentation` (Windows only): Whether to
+  install/manage [SignalFx Auto Instrumentation for .NET](
+  https://docs.splunk.com/Observability/gdi/get-data-in/application/dotnet/get-started.html).
+  When set to `true`, the `signalfx-dotnet-tracing` MSI package will be
+  downloaded and installed, and the Windows registry will be updated based on
+  the options below. (**default:** `false`)
+
+- `signalfx_dotnet_auto_instrumentation_version` (Windows only): Version of the
+  `signalfx-dotnet-tracing` MSI package to download and install from
+  [GitHub Releases](https://github.com/signalfx/signalfx-dotnet-tracing/releases).
+  By default, a request will be made to
+  `https://api.github.com/repos/signalfx/signalfx-dotnet-tracing/releases/latest`
+  to determine the latest release. If a version is specified, e.g. `1.0.0`,
+  the API request will be skipped and the MSI package will be downloaded from
+  `https://github.com/signalfx/signalfx-dotnet-tracing/releases/download/v{{ signalfx_dotnet_auto_instrumentation_version }}/signalfx-dotnet-tracing-{{ signalfx_dotnet_auto_instrumentation_version }}-x64.msi`.
+  (**default:** `latest`)
+
+- `signalfx_dotnet_auto_instrumentation_msi_url` (Windows only): Specify the
```

Comment:
I'd rewrite it as "Specify the URL to download the `signalfx-dotnet-tracing` MSI to skip..."